.github/workflows/prepare-for-prod-dt-deploy.yml

```diff
@@ -57,6 +67,9 @@ jobs:
       load_target: development
       push_target: production
       build_type: dt
+      use_merge_sha: true
+      skip_tests: ${{startsWith(github.event.pull_request.head.ref, 'hotfix-release/')}}
```

Comment:
we should be able to get `skip_tests` as one of the parameter while triggering github action too - can consider this as enhancement. So that release owner has control on hotfix releases.